### ISSUE TITLE
loss

### ISSUE THREAD

**Issue by YangYangGirl** [chenyuntc/simple-faster-rcnn-pytorch#142]

**Comment by YangYangGirl** [comment]
![image](https://user-images.githubusercontent.com/32426369/61171338-c2a20e80-a5a8-11e9-9412-f2d8047ac604.png)

**Comment by YangYangGirl** [comment]
I wrote the poink-linking-network based on your code,but the loss looked strange leaping after every epoch.So I had took a close look out at your loss,I found that there also was a relatively weak leaping after every epoch.https://github.com/YangYangGirl/point-linking-network

**Comment by 300LiterPropofol** [comment]
The leap is normal because the current repository only supports batch_size = 1 training,
it means it randomly select out one image instead of a mini batch that includes multiple images, this randomness causes the leap after every epoch, if you change the batch_size to 8 our 16, the leap will disappear.
